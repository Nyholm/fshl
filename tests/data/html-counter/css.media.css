@media (max-width: 960px) {
	#navigation {
		height: 52px;
	}
	#navigation ul:first-child + ul {
		clear: left;
		border: none;
		padding: 0;
	}
	.name, .attributes {
		white-space: normal;
	}
}

@media (max-width: 650px) {
	#search {
		float: left;
	}
	#navigation {
		height: 78px;
	}
	#navigation ul:first-child {
		clear: both;
	}
}