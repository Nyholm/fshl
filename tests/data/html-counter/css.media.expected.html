<html><head><meta http-equiv="content-type" content="text/html; charset=utf-8" /><link rel="stylesheet" type="text/css" href="../../../style.css" media="all" /></head><body><pre>
<span class="line"> 1: </span><span class="css-at-rule">@media (</span><span class="css-property">max-width</span><span class="css-at-rule">:</span><span class="css-value"> 960px</span><span class="css-at-rule">)</span> {
<span class="line"> 2: </span>    #<span class="css-property">navigation {
</span><span class="line"> 3: </span><span class="css-property">        height</span>:<span class="css-value"> 52px</span>;
<span class="line"> 4: </span>    }
<span class="line"> 5: </span>    <span class="css-id">#navigation</span> <span class="css-tag">ul</span><span class="css-pseudo">:first-child</span> + <span class="css-tag">ul</span> {
<span class="line"> 6: </span>        <span class="css-property">clear</span>:<span class="css-value"> left</span>;
<span class="line"> 7: </span>        <span class="css-property">border</span>:<span class="css-value"> none</span>;
<span class="line"> 8: </span>        <span class="css-property">padding</span>:<span class="css-value"> 0</span>;
<span class="line"> 9: </span>    }
<span class="line">10: </span>    <span class="css-class">.name</span>, <span class="css-class">.attributes</span> {
<span class="line">11: </span>        <span class="css-property">white-space</span>:<span class="css-value"> normal</span>;
<span class="line">12: </span>    }
<span class="line">13: </span>}
<span class="line">14: </span>
<span class="line">15: </span><span class="css-at-rule">@media (</span><span class="css-property">max-width</span><span class="css-at-rule">:</span><span class="css-value"> 650px</span><span class="css-at-rule">)</span> {
<span class="line">16: </span>    #<span class="css-property">search {
</span><span class="line">17: </span><span class="css-property">        float</span>:<span class="css-value"> left</span>;
<span class="line">18: </span>    }
<span class="line">19: </span>    <span class="css-id">#navigation</span> {
<span class="line">20: </span>        <span class="css-property">height</span>:<span class="css-value"> 78px</span>;
<span class="line">21: </span>    }
<span class="line">22: </span>    <span class="css-id">#navigation</span> <span class="css-tag">ul</span><span class="css-pseudo">:first-child</span> {
<span class="line">23: </span>        <span class="css-property">clear</span>:<span class="css-value"> both</span>;
<span class="line">24: </span>    }
<span class="line">25: </span>}
</pre></body></html>